From f0e33d92b0fac09ea21fd74f85ec15728982ebed Mon Sep 17 00:00:00 2001
From: "Richard W.M. Jones" <rjones@redhat.com>
Date: Thu, 5 Dec 2024 13:35:24 +0000
Subject: [PATCH] docs: Embolden whole "-O -"

Since this is typed literally make sure the whole string is
emboldened, according to our usual style.

(cherry picked from commit 3b81441fdb9e43eae3619c247e0d945d143fc92f)
(cherry picked from commit 66a6afb741d32f998633b121c2fb9813e5195820)
---
 docs/virt-v2v-inspector.pod | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/virt-v2v-inspector.pod b/docs/virt-v2v-inspector.pod
index 6b2ba2a9..8201fcf9 100644
--- a/docs/virt-v2v-inspector.pod
+++ b/docs/virt-v2v-inspector.pod
@@ -138,7 +138,7 @@ Display help.
 
 Write the output to a file called F<output.xml>.
 
-=item B<-O> -
+=item B<-O ->
 
 Write the output to stdout.  This is also the default if the I<-O>
 option is omitted.
